OpenStack-nova-accounting/extract_accounting.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
```

Comment:
Could we explicitly mark this as Python 3 if it's compatible, to save me checking it later?

Reply:
I've added comments about what needs to be done